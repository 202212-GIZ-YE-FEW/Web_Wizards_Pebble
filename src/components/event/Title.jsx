const Title = ({ text }) => {
    return (
        <h2 className='text-2xl font-sans text-black-100  font-semibold mb-2 '>
            {text}
        </h2>
    );
};

export default Title;
