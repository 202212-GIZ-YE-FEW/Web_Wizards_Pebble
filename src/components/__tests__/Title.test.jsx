import renderer from "react-test-renderer";

import Title from "../event/Title";

it("renders correctly", () => {
    const tree = renderer.create(<Title />).toJSON();
    expect(tree).toMatchSnapshot();
});
