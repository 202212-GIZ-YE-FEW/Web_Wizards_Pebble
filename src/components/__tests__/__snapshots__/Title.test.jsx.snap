// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2
  className="text-2xl font-semibold mb-2 "
/>
`;
